docs/StardustDocs/topics/createDataFrame.md

```diff
@@ -44,6 +44,23 @@ val df = dataFrameOf(
 
 <!---END-->
 
+Create DataFrame with nested columns inplace:
+
+<!---FUN createNestedDataFrameInplace-->
+
+```kotlin
+// DataFrame with 2 columns and 3 rows
+val df = dataFrameOf(
+    "name" to columnOf(
+        "firstName" to columnOf("Alice", "Bob", "Charlie"),
```

Comment:
Cool nested tree, I'd like this, seems like DSL could be used here also

Reply:
We could try to design such DSL after 1.0, as a better alternative to existing DataFrameBuilder functions like fill, withValues, etc, etc